decidim-admin/app/assets/javascripts/decidim/admin/officializations.js.es6

```diff
@@ -0,0 +1,20 @@
+$(() => {
+  const $modal = $("#showEmailModal");
```

Comment:
I'd suggest to use _kebab-case_ (`show-email-modal`) or _snake_case_ (`show_email_modal`) for `id`s and `class`es used in element attributes, and _camelCase_ is just fine for js ☺️ .

But I'm not sure there's a convention on this in Decidim.